### PR TITLE
Added early anbox instructions

### PR DESCRIPTION
**Pull request by NeoTheThird**

### CODE REVIEW

File: userguide/dailyuse/anbox.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.. note::
+    - Anbox is in early development
+    - Anbox for Ubuntu Touch is in even more early development
+    - Anbox only works on the 16.04 version of Ubuntu Touch, which is also in early development
+    - The only device currently supported is the Meizu Pro 5 ("turbo"), more to come in the near future
+
+How to install
+--------------
+
+Because this is only recommended for experienced users, installation instructions are intentionally left vague at the moment. With the software becoming more mature,
```
Comment (UniversalSuperBox) [inline userguide/dailyuse/anbox.rst]: You accidentally an ending to this sentence

File: userguide/dailyuse/anbox.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.. warning::
+    Because this feature is in such an early stage of development, the installation is only recommended for experienced users.
+
+- Install the 16.04/devel channel on your Meizu Pro 5 ("turbo")
+- flash http://cdimage.ubports.com/anbox-images/anbox-boot-turbo.img
```
Comment (UniversalSuperBox) [inline userguide/dailyuse/anbox.rst]: use the two-colons syntax here to create a code block:
```
- flash image::

    sudo touch ...
    other commands
    more commands...

- continued list
```

**Comment by nfsprodriver** [comment]
There should be a description, how to flash the boot.img, like:
Bring your device to fastboot, connect via fastboot on your computer and flsdh using: flash anbox-boot.img boot (don't know the right syntax currently)...

**Comment by UniversalSuperBox** [comment]
@nfsprodriver, I think the Fastboot instructions are left vague on purpose.

**Comment by NeoTheThird** [comment]
@nfsprodriver I (or someone else) can update the instructions with more detail later.

File: userguide/dailyuse/anbox.rst (hunk 2 above)
Comment (NeoTheThird) [reply userguide/dailyuse/anbox.rst]: Not sure if this makes sense. The commands are not related to flashing the image...

**Comment by NeoTheThird** [comment]
Please review again @nfsprodriver @UniversalSuperBox @mariogrip.

**Comment by nfsprodriver** [comment]
Unfortunately I don't have a device I can test it with, but the instructions look good now ;)

**Pull request merged**